src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -430,9 +127,42 @@ std::shared_ptr<ov::Model> FrontEnd::convert(const ov::frontend::InputModel::Ptr
         return function;
     }
 
+    auto translator_map = std::make_shared<TranslatorDictionaryType>();
+    auto default_translator_map = tensorflow::op::get_supported_ops();
+    translator_map->insert(default_translator_map.begin(), default_translator_map.end());
+
+    // register custom conversion extensions
+    if (!m_conversion_extensions.empty()) {
```

Comment:
Because you removed `translator_map` from `FrontEnd` members you have to insert extensions every time, why you removed it? I think it can be as it was before a parameter of `FrontEnd` but a copy (or reference) can be passed to `TranslateSession`.

Reply:
Recovered